tools/bin/3d-tiles-tools.js

```diff
@@ -250,6 +257,29 @@ function readGlbWriteB3dm(inputPath, outputPath, force) {
         });
 }
 
+function readGlbWriteI3dm(inputPath, outputPath, force) {
+    outputPath = defaultValue(outputPath, inputPath.slice(0, inputPath.length - 3) + 'i3dm');
+    return fileExists(outputPath)
+        .then(function(exists) {
+            if (!force && exists) {
+                console.log('File ' + outputPath + ' already exists. Specify -f or --force to overwrite existing file.');
+                return;
+            }
+            return fsReadFile(inputPath)
+                .then(function(data) {
+                    // Set I3dm spec requirements
+                    var featureTable = { INSTANCES_LENGTH : 0,
+                                         EAST_NORTH_UP : true,
+                                         POSITION : { byteOffset : 0 }
+                                       };
```

Comment:
Sorry... I take that back with EAST_NORTH_UP. That's a good value to set if the instance positions are located on the globe to orient them property, but for this the position is at zero and so EAST_NORTH_UP shouldn't be set.